collector.go

```diff
@@ -67,6 +74,29 @@ var (
 			"sv_tested":  {GAUGE, "server_testing_connections", 1, "Server connections currently running either server_reset_query or server_check_query, shown as connection"},
 			"sv_login":   {GAUGE, "server_login_connections", 1, "Server connections currently in the process of logging in, shown as connection"},
 			"maxwait":    {GAUGE, "client_maxwait_seconds", 1, "Age of oldest unserved client connection, shown as second"},
+			"pool_mode":  {LABEL, "pool_mode", 1, "The pooling mode in use."},
+		},
+		"lists": {
+			"databases":     {GAUGE, "databases_total", 1, "Number of databases"},
+			"users":         {GAUGE, "users_total", 1, "Number of users"},
+			"pools":         {GAUGE, "pools_total", 1, "Number of pools"},
+			"free_clients":  {GAUGE, "free_clients_total", 1, "Number of free clients"},
+			"used_clients":  {GAUGE, "used_clients_total", 1, "Number of used clients"},
+			"login_clients": {GAUGE, "login_clients_total", 1, "Number of clients in login state"},
+			"free_servers":  {GAUGE, "free_servers_total", 1, "Number of free servers"},
+			"used_servers":  {GAUGE, "used_servers_total", 1, "Number of used servers"},
+			"dns_names":     {GAUGE, "dns_names_total", 1, "dns names"},
+			"dns_zones":     {GAUGE, "dns_zones_total", 1, "dns zones"},
+			"dns_queries":   {GAUGE, "dns_queries_total", 1, "dns queries"},
+			"dns_pending":   {GAUGE, "dns_pending_total", 1, "dns pending"},
+		},
+
+		"mem": {
+			"name":     {LABEL, "name", 1, "mem name"},
+			"size":     {GAUGE, "size_total", 1, "mem size"},
+			"used":     {GAUGE, "used_total", 1, "mem used"},
+			"free":     {GAUGE, "free_total", 1, "mem free"},
+			"memtotal": {GAUGE, "memtotal_bytes_total", 1, "mem total"},
```

Comment:
Again, these should not end in `_total`. Also these need to have an appropriate unit, like `bytes`.

Can you find out what these metrics actually mean? Looks like this comes from [an external library](https://github.com/libusual/libusual).

Reply:
I added the `bytes` unit to it and remove the `_total`

these metrics coming from `SHOW MEM;`, it's one of PgBouncer Show commands

Reply:
Yes, but what are they related to? How are they useful to monitoring? We prefer to add metrics that are documented well, otherwise it's just junk data nobody can use.

Reply:
I removed it because I'm unable to find any documentation to related to how to use it